lazy-pages/src/lib.rs

```diff
@@ -20,68 +20,142 @@
 //! In runtime data for contract wasm memory pages can be loaded in lazy manner.
 //! All pages, which is supposed to be lazy, must be mprotected before contract execution.
 //! During execution data from storage is loaded for all pages, which has been accesed.
-//! See also `handle_sigsegv`.
+//! See also `user_signal_handler`.
```

Comment:
```suggestion
//! See also `sys::user_signal_handler` in the source code.
```

Because this fn is private

Reply:
Done